### PR TITLE
Stress checks for setup-java v2

### PR DESCRIPTION
**Pull request by Praveen2112**

### CODE REVIEW

**Comment by hashhar** [comment]
@Praveen2112 You might want to split the steps into jobs instead. All steps of a job runs on the same machine so most likely the download would've already been cached.

**Comment by Praveen2112** [comment]
I'll try it as multiple steps. But from their code it looks like they are downloaded for each steps.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,8 +40,9 @@ jobs:
           sudo apt-get clean
           df -h
       - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
+          distribution: 'zulu'
```
Comment (electrum) [inline .github/workflows/ci.yml:95]: No need to quote here

**Comment by kokosing** [comment]
Thanks

**Pull request closed without merging**